src/Docs/Convert/WikiApiService.php

```diff
@@ -379,7 +379,7 @@ private function createCategory($titleEn, $seoEn, $titleDe, $seoDe, $parentCateg
                             'metaTitle' => '',
                             'metaDescription' => '',
                             'media' => null,
-                            'searchableInAllLanguages' => false,
```

Comment:
@mnaczenski this is the english data set. are sure it is correct?

Reply:
The english articles should always be searchable in all languages. So yes, this is intended.